compiler/rustc_lint_defs/src/builtin.rs

```diff
@@ -3207,42 +3206,6 @@ declare_lint! {
     "detects large moves or copies",
 }
 
-declare_lint! {
```

Comment:
When a lint is removed, it must be added to the removed lint list. (This is explained at the top of this file.) That seems to be missing from this PR.

Reply:
done now